packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -104,6 +105,17 @@ export const skeleton = async (
     )
   }
 
+  if (tagsString) {
+    const tags = tagsString.split(',')
+    feedItemsQb = feedItemsQb.where(
+      'post.tags',
+      '@>',
+      sql`CAST(${JSON.stringify(tags)} as jsonb)`,
+    ) // AND logic
+    // @ts-ignore TODO
+    // feedItemsQb = feedItemsQb.where('post.tags', '?|', sql`CAST(${JSON.stringify(tags)} as jsonb)`) // OR logic, doesn't work
```

Comment:
So using the `@>` JSONB operator works fine, but applies AND logic.

To apply OR logic, we need like `where <jsonb> ?| array['tag', 'tag2']`, but I'm struggling to produce that with Kysely.

Halp 😅